CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKRenderer.java

```diff
@@ -118,6 +118,20 @@ public void AddRenderable(CVKRenderable renderable) {
         }   
     }
     
+    
+    /**
+     * Called from the parent CVKVisualProcessor when the VisualManager is
+     * being destroyed.
+     */
+    public void Destroy() {
+        // TODO HYDRA: Not sure if this is the best method, can it cause a
```

Comment:
Doesn't look like there is.  If you are waiting on named semaphores or fences I think you can set a timeout, but the device only has one wait and it has no timeout value.  So amend your comment?

Reply:
Comment removed